src/components/[guild]/Onboarding/components/FloatingProgress.tsx

```diff
@@ -0,0 +1,20 @@
+import GuildCreationProgress from "components/create-guild/GuildCreationProgress"
+import { useOnboardingContext } from "./OnboardingProvider"
+
+const FloatingProgress = ({}) => {
```

Comment:
We only use this component in one place, so we might not even need it

Reply:
I found the reason I used a new component here. It uses the `DynamicOnboardingProvider` context. I can't use it yet a layer above.